shared/desktop/renderer/index.js

```diff
@@ -53,6 +53,42 @@ function setupAvatar() {
   initAvatarLoad(loadUserImageMap, loadTeamImageMap)
 }
 
+var timeoutID
+
+// Consider us inactive after a minute of no input
+// for the purpose of marking chat messages read
+function setTimeoutListeners() {
```

Comment:
Ideally if the app is in use we'd not have to have these listeners going on window all the time. So maybe we should expand this such that if one of these listeners fires, we actually set a timeout (like 5 minutes) and remove all the listeners (we mark ourselves as non-idle). 
Once that 5 minutes is up, we can re-add the listeners and a timeout (like 1 minute) and if the listeners fire, we do the the above, else we mark as idle (and we can leave the listeners at this point since our app shouldn't be competing to render anything)